mne_bids/config.py

```diff
@@ -2,6 +2,13 @@
 from mne import io
 from mne.io.constants import FIFF
 
+try:
+    from mne.io import read_raw_persyst
+except ImportError:
+    print('Reading Persyst files requires latest mne dev '
+          'version or 0.21 release.')
```

Comment:
this will be printed all the time even for users that don't care about persyst? we should not do this. You should have a error / message only if you need it.